src/components/PasswordReset/index.jsx

```diff
@@ -0,0 +1,114 @@
+import { Alert, Input } from "antd";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import "./index.css";
+
+const Login = (props) => {
+  const [values, setValues] = useState({ email: "" });
+
+  const handleChange = (key, value) => {
+    let newValues = Object.assign({}, values);
+    newValues[key] = value;
+    setValues(newValues);
+  };
+  const [errors, setErrors] = useState({});
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    let { email, password } = values;
+    let errors = {};
+    if (!validateEmail(email)) {
+      errors["email"] = "Email format error";
+    }
+    if (email.trim() === "") {
+      errors["email"] = "Please fill this field";
+    }
+
+    setErrors(errors);
+    if (Object.keys(errors).length > 0) {
+      return;
+    }
+    if (typeof props.onSubmit == "function") {
+      props.onSubmit(values);
+    }
+  };
+
+  const handleGoToRegister = () => {
+    if (typeof props.onGoToRegister == "function") {
+      props.onGoToRegister();
+    }
+  };
+
+  const handleGoToLogin = () => {
+    if (typeof props.onGoToLogin == "function") {
+      props.onGoToLogin();
+    }
+  };
+
+  const handleGoToPasswordReset = ()=>{
+    if (typeof props.goToPasswordReset == "function") {
+      props.goToPasswordReset();
```

Comment:
onGoToRegister, onGoToLogin but goToPasswordReset -- there is an inconsistency here

Reply:
this piece of code was removed from this file while I was reviewing the code,  since we're already in password reset page